jest.config.js

```diff
@@ -0,0 +1,179 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
```

Comment:
I'd favour removing these if we're not setting them. It'll quickly become out of date if not carefully maintained.